htdocs/install/mysql/migration/18.0.0-19.0.0.sql

```diff
@@ -218,3 +218,7 @@ ALTER TABLE llx_bom_bomline ADD COLUMN tms timestamp DEFAULT CURRENT_TIMESTAMP O
 UPDATE llx_c_type_contact SET element = 'stocktransfer' WHERE element = 'StockTransfer';
 
 UPDATE llx_c_units SET scale = 1 WHERE code = 'S';
+
+DELETE FROM llx_c_tva WHERE taux = 16;
```

Comment:
This may delete line for other countries.
You can instead do an update with 
update taux = 3 from llx_c_tva where fk_pays = 102 and taux = 16

Reply:
Sorry! Fixed